contrib/sip_proxy/filters/network/source/conn_manager.cc

```diff
@@ -330,7 +329,6 @@ void ConnectionManager::onEvent(Network::ConnectionEvent event) {
 }
 
 DecoderEventHandler& ConnectionManager::newDecoderEventHandler(MessageMetadataSharedPtr metadata) {
-  stats_.counterFromElements(methodStr[metadata->methodType()], "request_received").inc();
```

Comment:
I can see methodStr is declared in `contrib/sip_proxy/filters/network/source/metadata.cc`

Can we construct a parallel vector of StatName in the Config object or something else that is instantiated either at process start, or when the filter is configured, with StatName views of those 12 items?

Also it would be better not to have to re-instantiate the counter by either name or StatName every time it is incremented.

Reply:
@jmarantz thanks for your comments.
I just moved back the stats change, and will record this comments and will consider it in next PR. Thanks.